ansible/roles/invoker/tasks/deploy.yml

```diff
@@ -25,9 +25,16 @@
     mode: 0777
   become: true
 
+- name: define options when deploying invoker on Linux
+  set_fact:
+    linuxOptions: "--pid=host --privileged -v /sys/fs/cgroup:/sys/fs/cgroup -v /run/runc:/run/runc -v /usr/bin/docker-runc:/usr/bin/docker-runc -v /usr/lib/x86_64-linux-gnu/libapparmor.so.1:/usr/lib/x86_64-linux-gnu/libapparmor.so.1"
+  when: whisk_version_name != "mac"
```

Comment:
is it possible to use runc w/o apparmor? then you could only put that into the linux_options